m4_sessions/1711-inning/Group_7.md

```diff
@@ -1,9 +1,46 @@
-Group Seven
+## Group Seven
 
-Session Title: Description: Why should students come to this session?
+### Session Title: 
+Be Your Own Ally
 
-Session Outcomes: What will students leave with or gain from this session?
+#### Description: 
 
-Session Outline: Include the flow of the session and the time for each section.
+#### Why should students come to this session?
+Turing and life are hard enough. If you are a minority, introverted, or just inexperienced with collaboration, it can be even harder. Women and other minorities in tech many times find themselves in difficult pairings.This session is designed for women and minorities in tech, especially those who don't feel comfortable speaking up for themselves. Participants will build a toolbox of strategies one can use to be a effective self-advocate in pairing and project teams.
 
-Names of group members:
+- how to advocate for yourself in pairing
+- how to respectfully make sure your voice is heard
+- how to address microaggressions
+- when to involve other people
+
+### Session Outcomes: 
+
+#### What will students leave with or gain from this session?
+- Participants will walk away with stratgies to productively address microagressions and any other interpersonal challenges that may arise from a pairing or team.
+- Participants will walk away with insight into issues that other students face and reflect on how to also advocate for others
+
+### Session Outline: 
+
+#### Include the flow of the session and the time for each section.
+Lola will intro video and facilitate icebreaker
+1. College Humor Video? - 4min
+2. Intro w/ icebreaker - 15 min
+  What is a common misconception about you? || Tell us about how you can relate to the video.
+  Read article https://www.psychologytoday.com/blog/happiness-purpose/201307/speak-yourself
+3.  Amy’s story- Amy gives the synopsis of her experience with the need to speak up for herself at Turing- 5 min
+
+Luke will intro video (if we use) and explain directions for #5
+4. Hook Video Clip- 10min if it’s past 1:25pm, skip video (last 7 minutes of Adam Galinsky vid)
+5. Scenarios and discussion- 30min
+	We’ll have four stations w/ a scenario for each and collect ideas on each scenario to share at the end of the exercise 
+(one station per facilitator)
+
+Hector will facilitate journaling and final reflection (we’ll all come up with journal prompts and have a synopsis of what to tell them to share for final reflection)
+6. Journaling- 5 min
+7. Final reflection- optional individuals shares takeaways - 10min
```

Comment:
Love this!!  I would be sure you each have talking points and that you discuss the scenarios before the session.  If each of you is at a station, then you can really push the conversation.